patches/addEduVulcanLoginButton.js

```diff
@@ -0,0 +1,4 @@
+if (!document.querySelector(".icon-logout"))
+  document.querySelector(".part-business").innerHTML =
+    '<a class="link-button button-primary" style="display: block;margin-top: 20px;" href="/logowanie">Zaloguj się</a>' +
+    document.querySelector(".part-business").innerHTML;
```

Comment:
using `document.createElement` and `insertBefore` seems like more proper way of doing it